e2e/test/FaultInjectionPoolAmqpTests.MessageSendFaultInjectionPoolAmqpTests.cs

```diff
@@ -821,27 +821,31 @@ public async Task Message_IoTHubSak_AuthenticationNoRecovery_MultipleConnections
                 Assert.IsTrue(isReceived, $"Message is not received for device {testDevice.Id}.");
             };
 
-            Func<IList<DeviceClient>, Task> cleanupOperation = async (deviceClients) =>
+            Func<IList<DeviceClient>, Task> cleanupOperation = (deviceClients) =>
             {
                 foreach (DeviceClient deviceClient in deviceClients)
                 {
                     deviceClient.Dispose();
                 }
+
+                return Task.FromResult(0);
             };
 
-            await FaultInjectionPoolingOverAmqp.TestFaultInjectionPoolAmqpAsync(
-                MessageSend_DevicePrefix,
-                transport,
-                poolSize,
-                devicesCount,
-                faultType,
-                reason,
-                delayInSec,
-                durationInSec,
-                (d, t) => { return Task.FromResult<bool>(false); },
-                testOperation,
-                cleanupOperation,
-                authScope).ConfigureAwait(false);
+            await FaultInjectionPoolingOverAmqp
```

Comment:
minor: I don't know if this is public facing or not but async function name does not seem to be ending with async